### PR TITLE
Revoking an accessToken should not remove the credential from storage

### PR DESCRIPTION
**Pull request by mikenachbaur-okta**

### CODE REVIEW

File: Samples/AuthenticationFlows.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ func signInWithWebUsingCustomConfiguration() async throws {
                                  redirectUri: redirectUrl)
     
     // Sign in using the above configuration
-    let token = try await auth.start(from: view.window)
+    let token = try await auth.signIn(from: view.window)
```
Comment (rajdeepnanua-okta) [inline Samples/AuthenticationFlows.swift:41]: How does this change the behavior?

**Review by rajdeepnanua-okta — APPROVED** [review]
Looks good. Had a question about the change in sample

File: Samples/AuthenticationFlows.swift (hunk 1 above)
Comment (mikenachbaur-okta) [reply Samples/AuthenticationFlows.swift:41]: This is just sample code, but a long time ago when I renamed the `start` method to `signIn`, I forgot to update the sample.

Comment (rajdeepnanua-okta) [reply Samples/AuthenticationFlows.swift:41]: Ah got it. Thanks for clarifying!

**Pull request merged**